### PR TITLE
docs: cleanup publishers

### PR DESCRIPTION
**Pull request by georgexu99**

### CODE REVIEW

File: config/publishers/snapcraft.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ Configuration options are documented in [`PublisherSnapConfig`](https://js.elect
 ```javascript
 {
   name: '@electron-forge/publisher-snapcraft',
-  config: {}
+  config: {
+    release: "1"
```
Comment (erickzhao) [inline config/publishers/snapcraft.md]: Should we add more usage details here?

File: config/publishers/bitbucket.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ Full configuration options are documented in [`PublisherBitbucketConfig`](https:
 
 ```javascript
 {
-  "name": "@electron-forge/publisher-bitbucket",
-  "config": {
-    "repository": {
-      "owner": "myusername",
-      "name": "myreponame"
+  name: "@electron-forge/publisher-bitbucket",
```
Comment (erickzhao) [inline config/publishers/bitbucket.md]: Three things here:
* We usually use single-quotes for JS
* Can we add the `forge.config.js` code snippet title? There's some GitBook-specific syntax that we can use here. See: https://github.com/electron-forge/electron-forge-docs/blob/v6/guides/code-signing/code-signing-macOS.md
* We might want to expand on this a bit by adding an abridged version of the full JS object in `forge.config.js`. Something like the below

```js
module.exports = {
  // ...
  publishers: [
     {
       name: 'asldjkas;d';ad",
       config: { }
     }
  ]

}
```

File: config/publishers/bitbucket.md [hunk 3]
```diff
@@ -16,15 +16,15 @@ Full configuration options are documented in [`PublisherBitbucketConfig`](https:
...
+  config: {
+    repository: {
+      owner: "myusername",
+      name: "myreponame"
     },
     "auth": {
```
Comment (erickzhao) [inline config/publishers/bitbucket.md]:
```suggestion
    auth: {
```

Forgot one here

**Review by erickzhao — CHANGES_REQUESTED** [review]

File: config/publishers/bitbucket.md (hunk 2 above)
Comment (georgexu99) [reply config/publishers/bitbucket.md]: good catch, in my fever to remove the quotes from the left side i forgot to look at the right

File: config/publishers/bitbucket.md [hunk 4]
```diff
@@ -14,18 +14,23 @@ Full configuration options are documented in [`PublisherBitbucketConfig`](https:
...
-  "name": "@electron-forge/publisher-bitbucket",
-  "config": {
-    "repository": {
-      "owner": "myusername",
-      "name": "myreponame"
-    },
-    "auth": {
-      "username": "myusername",
-      "appPassword": "mysecretapppassword"
+```javascript title=forge.config.js
```
Comment (erickzhao) [inline config/publishers/bitbucket.md]: `title=forge.config.js` isn't the correct syntax for Gitbook unfortunately

File: config/publishers/electron-release-server.md [hunk 5]
```diff
@@ -8,14 +8,19 @@ Configuration options are documented in [`PublisherERSConfig`](https://js.electr
...
+```javascript title=forge.config.js
+module.exports = {
+  // ...
+  publishers: [
+    {
+      name: '@electron-forge/publisher-electron-release-server',
+      config: {
+        baseUrl: 'https://update.server.com',
+        username: 'admin',
+        password: 'admin'
```
Comment (erickzhao) [inline config/publishers/electron-release-server.md]: Let's modify this so `password` doesn't appear to be plaintext.

**Pull request merged**